src/fides/api/schemas/tcf.py

```diff
@@ -14,40 +14,50 @@
 from fides.api.schemas.privacy_notice import UserSpecificConsentDetails
 
 
-class TCFSavedandServedDetails(UserSpecificConsentDetails):
-    """Default Schema that combines TCF details with whether a consent item was
-    previously saved or served."""
```

Comment:
nice, so this refactor should also make https://github.com/ethyca/fides/issues/4141 easier, right?

Reply:
Oh yes I hope so! Since I already had to separate out the default preferences for vendors/purposes based on legal basis, just went ahead and did them both while I was here!